atlasdb-jdbc/src/main/java/com/palantir/atlasdb/keyvalue/jdbc/impl/SingleTimestampPutBatch.java

```diff
@@ -42,6 +44,11 @@ public SingleTimestampPutBatch(Map<Cell, byte[]> data, Long timestamp) {
         this.timestamp = timestamp;
     }
 
+    public SingleTimestampPutBatch(List<Entry<Cell, byte[]>> data, long timestamp) {
+        this.data = data.stream().collect(Collectors.toMap(Entry::getKey, Entry::getValue));
+        this.timestamp = timestamp;
```

Comment:
I have no idea... I think I've needed this specific one at some point in the past though :)